commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/config/HazelcastHealthIndicator.java

```diff
@@ -0,0 +1,45 @@
+package io.mosip.commons.packetmanager.config;
+
+import java.lang.reflect.Method;
+
+import com.hazelcast.core.HazelcastInstance;
+
+import org.springframework.boot.actuate.health.AbstractHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+import org.springframework.util.ReflectionUtils;
+
+@Component
+@ConditionalOnProperty(value = "packetmanager.hazelcast.disable.health.check", matchIfMissing = true)
+public class HazelcastHealthIndicator extends AbstractHealthIndicator {
```

Comment:
Since hazelcast is a optional runtime dependency, should we have this indicator also loaded in runtime ?